elfpy/bots/budget.py

```diff
@@ -17,10 +17,26 @@ class Budget:
     max_wei: int = int(1e21)  # 1k ETH
 
     def sample_budget(self, rng: NumpyGenerator) -> FixedPoint:
-        """Returns a sample from a clipped normal distribution"""
+        """Returns a sample from a clipped normal distribution.
+
+        Parameters
+        ----------
+        rng : NumpyGenerator
+            Random number generator
+
+        Note
```

Comment:
note should go below Returns

Parameters should be Arguments

Reply:
I was looking at the [Numpy style guide](https://numpydoc.readthedocs.io/en/latest/format.html#parameters) last night. It seems to say Parameters, though I just realized we've been using Arguments this entire time. Didn't we want to follow the Numpy style?

for notes, I was looking for something to include below the first sentence. looks like they allow a paragraph there without a title. I guess that's what they refer to as the [Extended Summary](https://numpydoc.readthedocs.io/en/latest/format.html#extended-summary) which is meant to "clarify functionality, not to discuss implementation detail or background theory, which should rather be explored in the [Notes](https://numpydoc.readthedocs.io/en/latest/format.html#notes) section below"

Reply:
changed it to Arguments and using Extended Summary instead of Notes in 2c19b16decdd183149727b1b604d0a0a77a96cee